pkg/commands/app.go

```diff
@@ -1218,13 +1207,12 @@ func showVersion(cacheDir, outputFormat, version string, outputWriter io.Writer)
 
 	switch outputFormat {
 	case "json":
-		b, _ := json.Marshal(VersionInfo{
+		_ = json.NewEncoder(w).Encode(VersionInfo{
```

Comment:
error is ignored here, was it done intentionally ?

Reply:
That is not the code I added, and I have no idea why the error was ignored initially. But it looks like we should handle it. Fixed in https://github.com/aquasecurity/trivy/pull/4842/commits/1417f18582cb3a16edeccc42ccaef4abcfa09a58.